library/classes/Installer.class.php

```diff
@@ -146,7 +146,16 @@ public function drop_database() {
   }
 
   public function grant_privileges() {
+	$version =  mysqli_get_server_version($this->dbh);
+	if($version >= 80000){
+		$this->execute_sql("drop user $this->login@$this->loginhost");
+		$this->execute_sql("FLUSH PRIVILEDGES");
+		return ( ($this->execute_sql("CREATE USER '$this->login'@'$this->loginhost' IDENTIFIED with mysql_native_password  by '$this->pass'"))
+		&&($this->execute_sql("GRANT ALL PRIVILEGES ON $this->dbname.* TO '$this->login'@'$this->loginhost'")));
+	}
+	else{
     return $this->execute_sql( "GRANT ALL PRIVILEGES ON $this->dbname.* TO '$this->login'@'$this->loginhost' IDENTIFIED BY '$this->pass'" );
+		}
   }
```

Comment:
This is vulnerable to sql injection...

Reply:
@MathurinNkenfack address this properly and also consider trying this with docker by replacing the appropriate files with the necessary codes

Reply:
If you need help feel free to reach out

Reply:
I tried as much as could but to no avail that's why I instead used mysql 5 and secured the installing using prepared statements  in the following pull request #1586